docs/secure-guardrails/overview.md

```diff
@@ -0,0 +1,261 @@
+---
```

Comment:
This doc was copied directly from the `s-santillan` fork except for 1 change (I will mark).